src/constructorio-id.js

```diff
@@ -61,6 +63,12 @@
       if (this.cookie_domain) {
         cookie_data += '; domain=' + this.cookie_domain;
       }
+      if (this.cookie_secure) {
+        cookie_data += '; secure';
+      }
+      if (this.cookie_samesite) {
+        cookie_data += '; samesite=' + this.cookie_samesite;
```

Comment:
This won't affect functionality but it seems it's a convention to write it as capitalized as `SameSite`

Reply:
Correct but that applies to all params (domain, path, expires) and we don't do that for the rest of the params, I was planning to keep it consistent with the rest of the file

Reply:
Then we're good 👍